### PR TITLE
Updated frontend to be more mobile friendly. Hopefully ...

### PR DESCRIPTION
**Pull request by RobustProgram**

### CODE REVIEW

**Comment by Chester-P** [comment]
Hey there, those are some really good changes, I think it did make it more mobile friendly!
Can you just check my comment and revert the change in run-api.sh and I'll merge this in and deploy it!

File: run-api.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-gunicorn run:app --daemon --log-file uplanner-api.log -w 4 -b localhost:5000
```
Comment (Chester-P) [inline run-api.sh]: Can you revert this change? I'm gonna need this script to deploy it on my server

Comment (Chester-P) [reply run-api.sh]: for dev test, we can just run python run.py rather using gunicorn at all btw

**Comment by RobustProgram** [comment]
I have reverted the changes to run-api.sh. I am just not sure what you meant by "Can you just check my comment".

**Review by Chester-P — APPROVED** [review]

**Comment by Chester-P** [comment]
I forgot to submit my review messages 😝

File: run-api.sh (hunk 1 above)
Comment (RobustProgram) [reply run-api.sh]: Ah I see your comment. Thanks for the heads up :)
